src/utils/utils.ts

```diff
@@ -12,11 +13,20 @@ type MetadataSchemaEntries<T extends (...args: any) => any> = {
 export function recMetadataSchemaHelper<T extends (...args: any) => any>(
   keyArray: ReadonlyArray<string>,
   valueType: T,
-  defaultValue: ReturnType<T>
+  defaultValue: ReturnType<T>,
+  isRequired: boolean,
 ) {
   const obj: MetadataSchemaEntries<T> = {};
   for (const keyName of keyArray) {
-    obj[keyName] = { type: valueType, default: defaultValue, required: true };
+    obj[keyName] = { type: valueType, default: defaultValue, required: isRequired };
   }
   return obj;
 }
+
+export function removeMongoObjectID<T>(records: Array<T & {_id?: mongoose.Types.ObjectId}>): void
```

Comment:
What is this getting used for?

Reply:
See below - though arguably the function might be unnecessary as it'd be possible to just do with a `.map` instead.

Reply:
Ya let's just use map or project instead of deleting properties.